src/datasets/utils/streaming_download_manager.py

```diff
@@ -149,8 +149,12 @@ def _extract(self, urlpath: str) -> str:
             return f"{protocol}://::{urlpath}"
 
     def _get_extraction_protocol(self, urlpath: str) -> Optional[str]:
+        # get inner file: zip://train-00000.json.gz::https://foo.bar/data.zip -> zip://data/train-00000.json.gz
```

Comment:
I think it should be

~~~
         # get inner file: zip://train-00000.json.gz::https://foo.bar/data.zip -> zip://train-00000.json.gz
~~~